docs/releases/1.11.rst

```diff
@@ -121,8 +121,8 @@ to:
 
 and passing those through at the point where you are calling ``get_url_parts`` on ``super`` (if applicable).
 
-See also: :meth:`wagtail.core.models.Page.get_url_parts`, :meth:`wagtail.core.models.Page.get_url`,
-:meth:`wagtail.core.models.Page.get_full_url`, and :meth:`wagtail.core.models.Page.relative_url`
+See also: `wagtail.core.models.Page.get_url_parts`, `wagtail.core.models.Page.get_url`,
+`wagtail.core.models.Page.get_full_url`, and `wagtail.core.models.Page.relative_url`
```

Comment:
`wagtail.core.models` has been moved to `wagtail.models`, and these methods are still documented

```suggestion
See also: :meth:`wagtail.models.Page.get_url_parts`, :meth:`wagtail.models.Page.get_url`,
:meth:`wagtail.models.Page.get_full_url`, and :meth:`wagtail.models.Page.relative_url`
```